docs/TransparentLogging.md

```diff
@@ -0,0 +1,280 @@
+   Log, and to request cryptographic proofs of the append-only properties
+   (inclusion proofs and consistency proofs).
+ - A write-transparent Log allows any external client to submit new entries to
+   the Log, and receive either an inclusion proof for the entry or a signed
+   promise that it will be incorporated in future.
+
+Allowing write-transparency is vital for encouraging an ecosystem to grow up
+around the Log(s), and helps to reassure external (read-transparency) users that
+no "filtering" has been applied to content before it even reaches the log.
+
+However, it's worth making clear: a write-transparent Log allows arbitrary
+people on the Internet to write data into your append-only Log – and that **data
+can't be removed<sup>[3](#note-3)</sup> without destroying the Log**.
+
+That's a sufficiently terrifying prospect to motivate having strict **admission
+criteria**: checks that submitted content has to pass before being included in
+the Log.  (However, this isn't specific to write-transparent Logs; even if
+submissions are restricted to a whitelist of clients, the admission criteria
+still need to be detailed.)
+
+Examples:
+
+ - For Certificate Transparency, each submission has to take the form of an
+   X.509 certificate in ASN.1 DER format, together with a chain of CA
+   certificates that leads to one of a set of accepted root certificates.
+ - For Binary Transparency, each submission might need to be of the correct
+   layout/format to allow installation on a device, and might need to be signed
+   by one of a set of acceptable package signing keys.
+ - For a Pastebin Log, the admission criteria might merely be a size limit, or a
+   UTF-8 encoding correctness check.
+
+These examples illustrate that good admission criteria typically include two key aspects:
+ - Structure: The submissions have to be of the correct format to be of interest
+   (so that the submissions match the subject of the Log), and that format has
+   to be machine-checkable (but need not be strict – see below).
+ - Authentication: The submissions are signed by one of a limited number of
+   private keys, so the signature can be checked with the corresponding public
+   key.  The Log is thus implicitly
+     - assuming that the holders of those private keys will not sign excessive
+       numbers of objects (and so allow the Log to be swamped)
+     - trusting the key holders not to sign anything that might be problematic
+       for the Log to host.
+
+For our examples:
+ - A Certificate Transparency log requires entries to have X.509 structure, and
+   trusted-roots authentication.
+ - A Binary Transparency log requires entries to have the structure of an
+   installable binary package, and vendor-signing key authentication.
+ - A Pastebin log has little structural requirement and no authentication
+   requirement, and so is vulnerable to different attacks (swamping, uploading
+   illegal content etc.)
```

Comment:
Nit: the first two are also vulnerable to these attacks too, it's just less likely since it's attributable to another entity due to the signature.

Reply:
added a "much more" before "vulnerable to".